controllers/extension-certificate-service/charts/cert-broker-rbac/templates/rbac.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Do you want to move these charts into a subfolder (like the others), e.g. `charts/internal/cert-broker`?